pkg/provisioner/resourcehandlers.go

```diff
@@ -433,7 +432,7 @@ func deleteObjectBucket(ob *v1alpha1.ObjectBucket, c versioned.Interface) error
 
 	logD.Info("removing ObjectBucket finalizer", "name", ob.Name)
 	removeFinalizer(ob)
-	ob, err := c.ObjectbucketV1alpha1().ObjectBuckets().Update(context.TODO(), ob, metav1.UpdateOptions{})
+	_, err := c.ObjectbucketV1alpha1().ObjectBuckets().Update(context.TODO(), ob, metav1.UpdateOptions{})
```

Comment:
are you sure there is nothing added to the ob by `Update` that should be returned? I'm thinking of UID, timestamp, revision fields, etc...

Reply:
Created https://github.com/kube-object-storage/lib-bucket-provisioner/issues/202 for this.

Reply:
Well, the next step is to delete the OB, which is done with `ob.Name` and none of the other ob fields.